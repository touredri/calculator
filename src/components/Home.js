import React from 'react';

export default function Home() {
  return (
    <div className="home">
      <h2>Welcome to our page</h2>
      <div style={{ padding: '50px' }}>
        Welcome to our math website! Here, you can easily perform various
        mathematical operations and calculations, whether you&apos;re a student, a
        teacher, or just someone who loves math. Our user-friendly interface
        makes it easy to input equations and get accurate results in no time.
        Whether you need to solve equations, graph functions, or perform complex
        calculations, we&apos;ve got you covered. So, let&apos;s get started and explore
        the exciting world of mathematics together!
      </div>
    </div>
  );
}
