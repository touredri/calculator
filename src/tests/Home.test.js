import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/Home';

describe('Home', () => {
  it('renders the text', () => {
    const { getByText } = render(<Home />);
    expect(getByText('Welcome to our page')).toBeInTheDocument();
  });

  it('renders correct', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
