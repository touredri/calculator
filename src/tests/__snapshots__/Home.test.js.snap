// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correct 1`] = `
<div>
  <div
    class="home"
  >
    <h2>
      Welcome to our page
    </h2>
    <div
      style="padding: 50px;"
    >
      Welcome to our math website! Here, you can easily perform various mathematical operations and calculations, whether you're a student, a teacher, or just someone who loves math. Our user-friendly interface makes it easy to input equations and get accurate results in no time. Whether you need to solve equations, graph functions, or perform complex calculations, we've got you covered. So, let's get started and explore the exciting world of mathematics together!
    </div>
  </div>
</div>
`;
